inst/examples/app/analytics/R/prepare-admin-panel.R

```diff
@@ -786,6 +649,7 @@ prepare_admin_panel_components <- function(
   # sessions stats
 
   sessions_data <- shiny::reactive({
+    shiny::req(!is.null(session_details()))
```

Comment:
This is related to session log files that are not available. I think generally, this is a good check to make.